test/seedsplit.js

```diff
@@ -37,7 +37,7 @@ describe('seedsplit', () => {
     shardsList3 = await results[2]
     shardsList4 = await results[3]
 
-    return Promise.all(results)
+    await Promise.all(results)
```

Comment:
I am not sure if awaiting at the end of an async function makes any difference, but I don't mind.

Reply:
it's pretty much the same. I just thought it didn't make sense to return something in a function which isn't used by some other function :)

Reply:
Okay! Looks good to me.